src/store/channels/saga.ts

```diff
@@ -62,9 +90,19 @@ export function* clearChannels() {
   yield put(removeAll({ schema: schema.key }));
 }
 
+function* listenForChannelLoadedEvent() {
+  const channel = yield call(conversationsChannel);
+  while (true) {
+    const { channelId = undefined } = yield take(channel, ChannelEvents.MessagesLoadedForChannel);
```

Comment:
Wonder if we can use the `takeEveryFromBus` helper that I created a couple weeks ago?

Reply:
Works! Thankss